doc/man3/OPENSSL_hexchar2int.pod

```diff
@@ -26,6 +26,8 @@ equivalent.
 
 OPENSSL_hexstr2buf_ex() decodes the hex string B<str> and places the
 resulting string of bytes in the given I<buf>.
+The character I<sep> is the separator between the bytes, which is normally ':',
```

Comment:
What does normally mean in this context. There is some default but it applies only the original function without _ex. So perhaps it would make sense to mention this ':' there.

Reply:
Normally would indicate that this is the value that you are most likely to use here.. I have removed it, and moved the comment into hexstrtobuf.